ansible/config/linux/docker_dnsmasq_conf

```diff
@@ -0,0 +1,19 @@
+# Set dns forwarder.
```

Comment:
I left this file untouched because I remembered that you told me so but I could be wrong, I can delete it if it seems ok